python/cuml/test/test_fil.py

```diff
@@ -419,6 +419,23 @@ def test_output_blocks_per_sm(storage_type, blocks_per_sm,
     assert np.allclose(fil_preds, xgb_preds_int, 1e-3)
 
 
+@pytest.mark.skipif(has_xgboost() is False, reason="need to install xgboost")
+def test_print_forest_shape(small_classifier_and_preds):
+    model_path, model_type, X, xgb_preds = small_classifier_and_preds
+    import io
+    strio = io.BytesIO()
+    ForestInference.load(model_path, model_type=model_type, output_class=True,
+                         forest_shape_file=strio)
+    strio.seek(0)
+    string = str(strio.read())
```

Comment:
Might as well make my comment here, this way of getting the information is very unpythonic and will feel not familiar to typical users, was wondering on your thoughts about modifying it a bit? Was thinking perhaps changing the parameter `forest_shape_file` to a boolean, and then store the shape info as an attribute of the ForestInference object as a string, what do you think?

Reply:
sounds good to me! @canonizer what do you think?
Also, should we name them `compute_shape` and `shape_str`?

Reply:
The boolean parameter and storing the result into an attribute sound fine to me. Regarding the names, `shape_str` is fine, but I would name the boolean parameter `compute_shape_str`.